shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -497,7 +497,10 @@ object GlutenConfig {
       ("spark.hadoop.input.read.timeout", "180000"),
       ("spark.hadoop.input.write.timeout", "180000"),
       ("spark.hadoop.dfs.client.log.severity", "INFO"),
-      ("spark.sql.orc.compression.codec", "snappy")
+      ("spark.sql.orc.compression.codec", "snappy"),
+      (
+        COLUMNAR_VELOX_FILE_HANDLE_CACHE_ENABLED.key,
+        COLUMNAR_VELOX_FILE_HANDLE_CACHE_ENABLED.defaultValueString)
```

Comment:
nit: config with `spark.gluten.sql.columnar.backend.velox` prefix will be passed default. so you dont need pass it explicitly.

Reply:
Thanks, we have default value defined in both here and `VeloxBackend`. I want the value here dose the trick, although the values are same.

Reply:
```
        COLUMNAR_VELOX_FILE_HANDLE_CACHE_ENABLED.key,
        COLUMNAR_VELOX_FILE_HANDLE_CACHE_ENABLED.defaultValueString)
```

I mean you do not need pass backend-prefix config from java to native, it will be passed automatically by below codes.
```
    conf
      .filter(_._1.startsWith(backendPrefix))
      .foreach(entry => nativeConfMap.put(entry._1, entry._2))
```

it just a small issue, not block merge this patch.